extensions/guacamole-auth-cas/pom.xml

```diff
@@ -254,6 +254,14 @@
             <version>2.5</version>
             <scope>provided</scope>
         </dependency>
+        
+        <!-- JUnit -->
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
I only see a unit test for guacamole-ext in these changes. Is this left over from `CASTokenName`?

Reply:
Yeah - removed here and in LDAP module.